test/convert.bats

```diff
@@ -64,3 +64,17 @@ EOF
   rm -f stacker.yaml stacker-subs.yaml
   stacker clean
 }
+
+@test "build python image" {
+  git clone https://github.com/docker-library/python.git
```

Comment:
Some day you'll wake up and this repository will have changed and tests will fail.  Is that desirable?
The alternative is to put the test you need here in the repository, which will be more reliable and faster, no?

Reply:
This was to make sure the **exact** upstream tests work.

Reply:
Is it better to fork and then use the fork?

Reply:
I can understand the desire to make sure that "real world examples" work.  I personally hate it when the internet breaks my c-i.  It means someone has to spend time figuring out why the light went from green to red, or worse, people accept "sometimes c-i breaks".

If you want to have a collection of "moving target real world docker files", maybe that is best to do in an integration test of some sort.  Unfortunately that just kicks the "figure out what failed" problem down the line.

It is entirely possible that a stacker change regresses this behavior and this test catches that.  But I think it is more likely that a breakage occurs due to the added dependency on a moving third party.

I would much prefer a quicker that did something like:

```
@test "check environment variable substitution" {
   cat > dockerfile <<EOF
FROM alpine:3.17

ENV PATH /usr/local/bin:$PATH

RUN [ "$PATH" = "/usr/local/bin:/usr/bin:/usr/sbin" ]
EOF
}
```

I like the REGISTRY_URL so that these tests don't have to depend on docker.io.

Reply:
That Dockerfile in that repo unlikely to change for a long while and if it breaks or more likely when it disappears, we will look at it and fix.

Reply:
> That Dockerfile in that repo unlikely to change for a long while and if it breaks or more likely when it disappears, we will look at it and fix.

I disagree with that statement.  It has [changed 5 times in 2 weeks](https://github.com/docker-library/python/commits/0a56f04a07495e0ed61663c73bc0c2e175773ddc/3.11/alpine3.17/Dockerfile) including a revert.

I really think it is very high cost in build time (we're building *python* as part of stacker c-i?) and reliability.  Successful c-i now depends on:
 * access to 'alpine:3.17' that will change over time
 * 'git clone' from github (which does not auto-retry on failure) and basically cannot be mirrored

And all of this is to check the expansion of a templated string.

Reply:
@smoser, moved the conversion out of the core ci path, and add a "nightly" conversion workflow, at least some place we can catch issues.